frontend/src/__tests__/cypress/cypress/tests/mocked/projects/workbench.cy.ts

```diff
@@ -149,7 +175,37 @@ describe('Workbench page', () => {
   });
 
   it('Create workbench', () => {
-    initIntercepts({ isEmpty: true });
+    initIntercepts({
+      isEmpty: true,
+      notebookSizes: [
```

Comment:
Is this necessary to pass `notebookSizes` in every tests? Can we just pass this only in required test?